wdl-lint/tests/lints/key-value-pairs/source.errors

```diff
@@ -0,0 +1,130 @@
+note[KeyValuePairs]: item should be followed by a newline
+  ┌─ tests/lints/key-value-pairs/source.wdl:8:22
+  │
+8 │         another_key: ["value1",
+  │                      ^
+  │
+  = fix: add a newline after this element
+
+note[KeyValuePairs]: incorrect indentation
+  ┌─ tests/lints/key-value-pairs/source.wdl:8:32
+  │  
+8 │           another_key: ["value1",
+  │ ╭───────────────────────────────^
+9 │ │         "value2", "value3"]
+  │ ╰────────^
+  │  
+  = fix: add 4 spaces to indentation
+
+note[KeyValuePairs]: item should be followed by a newline
+  ┌─ tests/lints/key-value-pairs/source.wdl:9:9
+  │
+9 │         "value2", "value3"]
+  │         ^^^^^^^^^^
```

Comment:
I don't think this reported span should include this whitespace

Reply:
OK, I made `find_next_comma()` a public crate function in the `TrailingComma` rule and imported it here. I am now using that to find the comma and running the `Span` until the end of the comma.